### PR TITLE
Add Nue Double GPO (au version)

### PR DESCRIPTION
**Pull request by mmnpkf**
Hi.  I've added the code to connect the AU version of the Neu Double Power Outlet (https://www.amazon.com.au/Approved-Standard-Nue-SmartThings-Automation/dp/B07GX943VP/ref=sr_1_15?ie=UTF8&qid=1538269827&sr=8-15&keywords=nue+zigbee).  This is my first ever pull request so if I've messed it up please let me know.  Thanks, Patrick

### CODE REVIEW

**Comment by ryanbeaton** [comment]
Sorry meant to say good job, thanks

**Comment by mmnpkf** [comment]
I see that my pull request has failed - “continuous-integrations/travis-ci/pr - The Travis CI build failed”. Can anyone tell me what this means?

**Comment by mmnpkf** [comment]
@ryanbeaton.  Can you please elaborate on your zigbeeModel comment.  What I’ve put in the code is what comes up when I pair the device...

**Comment by ryanbeaton** [comment]
see https://github.com/Koenkk/zigbee2mqtt/issues/388#issuecomment-424858076

0x342e3147484431544b532d3635424e46 = 4.1GHD1TKS-65BNF
Which flipped would be FNB56-SKT1DHG1.4

That look like the model numbers of the Nue kit that @highground88 had in https://github.com/Koenkk/zigbee2mqtt/issues/176

**Comment by ryanbeaton** [comment]
/home/travis/build/Koenkk/zigbee-shepherd-converters/converters/fromZigbee.js
  889:1  error  Trailing spaces not allowed  no-trailing-spaces
/home/travis/build/Koenkk/zigbee-shepherd-converters/devices.js
  1263:38  error  Trailing spaces not allowed  no-trailing-spaces
  1263:38  error  Missing trailing comma       comma-dangle
✖ 3 problems (3 errors, 0 warnings)

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -873,7 +873,20 @@ const converters = {
...
+            const mapping = {12: 'left', 11: 'right'};
+            const whichButton = mapping[msg.endpoints[0].epId];
+            if (whichButton) {
+                const payload = {};
+                payload[`state_${whichButton}`] = msg.data.data['onOff'] === 1 ? 'ON' : 'OFF';
+                return payload;
+            }
+        },
+    },
+
```
Comment (ryanbeaton) [inline converters/fromZigbee.js]: delete this line

File: devices.js [hunk 2]
```diff
@@ -1252,6 +1252,16 @@ const devices = [
...
     },
+    {
+        zigbeeModel: ['0x342e3147484431544b532d3635424e46'],
+        model: 'MG-AUWS01',
+        vendor: 'Nue',
+        description: 'ZigBee double power point',
+        supports: 'on/off',
+        fromZigbee: [fz.nue_power_state, fz.ignore_onoff_change],
+        toZigbee: [tz.onoff],
+        ep: {'left': 12, 'right': 11}
```
Comment (ryanbeaton) [inline devices.js]: remove five spaces and add comma

**Comment by highground88** [comment]
@ryanbeaton Good pickup! Except I don't have any GPO's ... Just various switches. Maybe the reversed ID is in fact what they've done with the outlet version?

**Comment by mmnpkf** [comment]
> see [Koenkk/zigbee2mqtt#388 (comment)](https://github.com/Koenkk/zigbee2mqtt/issues/388#issuecomment-424858076)
> 
> 0x342e3147484431544b532d3635424e46 = 4.1GHD1TKS-65BNF
> Which flipped would be FNB56-SKT1DHG1.4
> 
> That look like the model numbers of the Nue kit that @highground88 had in [Koenkk/zigbee2mqtt#176](https://github.com/Koenkk/zigbee2mqtt/issues/176)

Sorry, I'm a total Noob.  Is this a bug in what I've done - that is, can I fix it?  I tried updating the Zigbee model number in my code to the "flipped" text that you mention in your comment but then when I start zigbee2mqtt it says that my GPO is an unknown device.  It only works when I use the long string.

**Comment by mmnpkf** [comment]
@ryanbeaton Thank you for the feedback about the spaces and comma.  I've fixed this and the check seems to be happy now.  Thanks!

**Comment by ryanbeaton** [comment]
Hi @mmnpkf 
in https://github.com/Koenkk/zigbee2mqtt/issues/388#issuecomment-425673166 @Koenkk said it was a bug in zcl-packet and he fixed it
in https://github.com/Koenkk/zigbee-shepherd-converters/pull/93#issuecomment-426176736 @Koenkk said update zigbee2mqtt and then repair device
Sorry that's all I know.

**Comment by Koenkk** [comment]
@mmnpkf fix it by:

- stopping zigbee2mqtt
- remove all entries of this device from `database.db` (there may be multiple)
- start zigbee2mqtt
- pair device

**Comment by mmnpkf** [comment]
Thanks @ryanbeaton and @Koenkk, your suggested fix worked!  Thank you @ryanbeaton for your patience and guidance.

I had some problems with github last night. I made the change to the zigbee model number but it just wouldn’t show here in the pull request.  However, it is showing this morning - hooray!

Yes, @Koenkk this is now ready to be merged, thank you.

Patrick

**Comment by Koenkk** [comment]
@mmnpkf nice, thanks!

**Pull request merged**